tox.ini

```diff
@@ -156,7 +156,8 @@ allowlist_externals = bash
 commands =
     # explicitly pass this as a bash command to set PST_VERSION
     bash -c "PST_VERSION=$(pip show pydata-sphinx-theme | grep Version | awk -F': ' '{print $2}') && \
-    pybabel extract . -F babel.cfg -o src/pydata_sphinx_theme/locale/sphinx.pot --project=pydata-sphinx-theme --copyright-holder='PyData developers' --version=$PST_VERSION"
+    pybabel extract . -F babel.cfg -o src/pydata_sphinx_theme/locale/sphinx.pot --keywords='_ __ l_ lazy_gettext' \
```

Comment:
this just makes the keywords explicit (added them while debugging, but there is no harm in leaving them)